pyodide-build/pyodide_build/common.py

```diff
@@ -209,3 +171,30 @@ def get_make_environment_vars():
             value = value.strip("'").strip()
             environment[varname] = value
     return environment
+
+
+def search_pyodide_root(base: str | Path, depth: int, max_depth: int = 5) -> Path:
```

Comment:
```suggestion
def search_pyodide_root(base: str | Path, depth: int = 0, max_depth: int = 5) -> Path:
```

Reply:
Probably for the type system it is convenient to use a helper function for the recursion...